gcpy/util.py

```diff
@@ -1646,7 +1646,9 @@ def get_filepath(
         col,
         date,
         is_gchp=False,
-        gchp_format_is_legacy=False
+        gchp_res="00",
+        gchp_is_pre_13.1=False,
+        gchp_is_pre_14.0=False
```

Comment:
Can you change the variable names to not include a period? It makes it confusing as to whether you are accessing an object attribute. Maybe `gchp_is_pre_13_1`?

Reply:
Same as above.